api/client.go

```diff
@@ -32,9 +32,7 @@ type ClientConfig struct {
 // NewClient returns a client to make api requests
 func NewClient(c *ClientConfig, httpClient httpClient) *Client {
 	if httpClient == nil {
-		httpClient = &http.Client{
-			Timeout: time.Second * 5,
-		}
+		httpClient = &http.Client{}
```

Comment:
I was just curious if you were seeing that the 5 second timeout wasn't enough for this driver?

Reply:
Yeah it wasn't enough to run through a full init and plan when the task had started off disabled.

Reply:
Good to know! Thank you